d2l-typography.html

```diff
@@ -1,9 +1,10 @@
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../d2l-colors/d2l-colors.html">
+<link rel="import" href="d2l-typography-shared-styles.html">
 <link href="https://s.brightspace.com/lib/fonts/0.4.0/fonts.css" rel="stylesheet">
 <dom-module id="d2l-typography">
 	<template>
-		<style>
+		<style include="d2l-typography-shared-styles d2l-colors">
```

Comment:
The color variables are defined in `html` in [d2l-colors.html](https://github.com/BrightspaceUI/colors/blob/master/d2l-colors.html) so only the import should be needed.